app/src/main/java/com/plenart/organizeme/adapters/LabelColorListAdapter.kt

```diff
@@ -7,7 +7,6 @@ import android.view.View
 import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
 import com.plenart.organizeme.databinding.ItemLabelColorBinding
-import com.plenart.organizeme.interfaces.CardItemClickInterface
 import com.plenart.organizeme.interfaces.LabelColorClickedInterface
 
 class LabelColorListAdapter(private val context: Context,private val list: ArrayList<String>, private val mSelectedColor: String)
```

Comment:
Check [comment]( https://github.com/perooo1/OrganizeMe-App/pull/2/files#r728087183). Context is unused so you can remove it